examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -141,37 +142,33 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
         mainAxisAlignment: MainAxisAlignment.start,
         children: <Widget>[
           new ListItem(
-            title: new Text('Scrollable dropdown:'),
+            title: new Text('Simple dropdown:'),
             trailing: new DropdownButton<String>(
-              value: dropdown1Value,
+              value: dropdown2Value,
```

Comment:
nit: the order now is that the values are used 2,3,1. It would be cleaner if the numbers matched the order they were in the code.